go.mod

```diff
@@ -3,11 +3,11 @@ module github.com/cortexproject/cortex-tools
 go 1.22.2
 
 require (
-	cloud.google.com/go/storage v1.10.0
```

Comment:
This version has been retracted due to https://github.com/googleapis/google-cloud-go/issues/6857. Let's use a newer version that fixes the bug, e.g. `v1.40.0`

Reply:
it's the version used by cortex v1.14.1 https://github.com/cortexproject/cortex/blob/v1.14.1/go.mod#L78
will not touch this